src/concepts/components/emitevents.md

```diff
@@ -0,0 +1,334 @@
+---
+description: Component could share update to the parent html component emit events
+---
+
+# Emit events
+
+It's possible to update the parent containers emitting **events** by the same way we received data via **properties**.
+
+The events are defined as part of the `Properties` struct.
+
+When it's needed you just need to call the `emit()` method of the event, to propagate the update to the parents containers.
```

Comment:
I don't think "to the parents containers" sounds entirely correct.

Reply:
I would change to "parent HTML element"